src/prelaunch_data_generator/cl_validator_keystores/cl_validator_keystore_generator.star

```diff
@@ -71,8 +70,20 @@ def generate_cl_validator_keystores(
 
 	# Store outputs into files artifacts
 	keystore_files = []
-	for idx, output_dirpath in enumerate(all_output_dirpaths):
-		artifact_name = plan.store_service_files(service_name, output_dirpath, name = "validator-keystore-" + str(idx+1))
+	for idx, participant in enumerate(participants):
+		output_dirpath = all_output_dirpaths[idx]
+
+		padded_idx = zfill_custom(idx+1, len(str(len(participants))))
```

Comment:
any reason this is len(str(len instead of just len(participants)?
`padded_idx = zfill_custom(idx+1, (len(participants))` should still work.

Reply:
Really? 🤔 
So, if participants has 13 elements, let's say
`zfill_custom(1, (len(participants))` would be `zfill_custom(idx+1, 13)` => `0000000000001`
whereas
`zfill_custom(idx+1, len(str(len(participants))))` would be `zfill_custom(idx+1, 2)` => `01`

I believe the second one is better, right? Maybe I'm confused, am i missing something?

Reply:
Right my bad. 
I just used another way to write this, but this can also work indeed. Your way is actually a lot nicer haha. https://github.com/kurtosis-tech/eth-network-package/blob/main/src/participant_network.star#L244